Calculator/Date_Object.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Net.NetworkInformation;
+
+namespace Calculator
+{
+    public class Date_Object
+    {
+        private readonly IDNS _dNS;
+
+        public Date_Object(IDNS dNS)
```

Comment:
Prefer using primary constructor / primary constructor with fields if we are only assigning things inside constructor and not calculating anything.